api/src/pcapi/routes/backoffice/templates/accounts/update_requests_list.html

```diff
@@ -74,12 +74,22 @@ <h2 class="fw-light">
                                    data-bs-target="#accept-{{ update_request.dsApplicationId }}">Accepter le dossier</a>
                               </li>
                             {% endif %}
-                          </ul>
-                        </div>
+                          {% endif %}
+                          {% if update_request.can_be_corrected %}
+                            <li class="dropdown-item p-0">
+                              <a class="btn btn-sm d-block w-100 text-start px-3"
+                                 data-bs-toggle="modal"
+                                 data-bs-target="#ask-for-correction-{{ update_request.dsApplicationId }}">Faire une demande de correction</a>
+                            </li>
+                          {% endif %}
+                        </ul>
                       </div>
-                      {% if update_request.status == update_request.status.on_going and update_request.user is not none %}
-                        {{ build_lazy_modal(url_for("backoffice_web.account_update.get_accept_form", ds_application_id=update_request.dsApplicationId) , "accept-" + update_request.dsApplicationId|string, "lazy", "xl") }}
-                      {% endif %}
+                    </div>
+                    {% if update_request.status == update_request.status.on_going and update_request.user is not none %}
+                      {{ build_lazy_modal(url_for("backoffice_web.account_update.get_accept_form", ds_application_id=update_request.dsApplicationId) , "accept-" + update_request.dsApplicationId|string, "lazy", "xl") }}
+                    {% endif %}
+                    {% if update_request.status == update_request.status.on_going %}
```

Comment:
La condition devrait être la même qu'au-dessus, non ?
```suggestion
                    {% if update_request.can_be_corrected %}
```